### PR TITLE
Hotfix 0.7.1

### PR DESCRIPTION
**Pull request by mstonefield**
Will resolve any conflicts once review is completed.

### CODE REVIEW

File: events/btr_sedriicae_leader__gender_event.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+### Sedriicae female leader only event
```
Comment (puckhead73) [inline events/btr_sedriicae_leader__gender_event.txt:1]: File name has a double '_'

Comment (mstonefield) [reply events/btr_sedriicae_leader__gender_event.txt:1]: You have a double

File: localisation/english/btr_archaeological_events_l_english.yml [hunk 2]
```diff
@@ -204,4 +204,23 @@
...
+ btr_site.152.desc:0 "After a significant amount of time and effort from our team the caverns have been reached. Our peoples experience with tunneling proved crucial in getting this project completed in a reasonable amount of time despite the difficulties with mining through solid diamond. As we had predicted, the chambers are in fact rooms clearly built by some alien civilization with technology that is largely unknown to us. The room has power with bright blue lights illuminating the strange, purple-gray metal that was used to build the room and also many of the machines inside of it. There is not much space despite the fact that the room is immense, because the majority of it is taken up by a massive wall of strange, triangular boxes protruding from an otherwise smooth wall. The boxes are oriented with one pointing up and another pointing down directly next to it with a small gap before the next pair. The boxes have alien symbols that glow with the blue light although we cannot understand them. Using the small hallway that runs along the strange wall our team has set up a new camp from which they will begin studying these boxes, the alien metals, and how this room is powered."
```
Comment (puckhead73) [inline localisation/english/btr_archaeological_events_l_english.yml:219]: "Our people's"

File: localisation/english/btr_archaeological_events_l_english.yml [hunk 3]
```diff
@@ -204,4 +204,23 @@
...
+ btr_site.153.name:0 "Long, Long Ago"
+ btr_site.153.desc:0 "With long hours of careful study, we have come to the conclusion that the strange boxes are in fact a form of cryostasis. A technology that has until now been purely hypothetical to our people. This has presented several challenges to our researchers. Due to the largely intact nature of the chamber it is very likely that whoever is inside these chambers is still alive, waiting to be reawakened. However, since we do not know why they went into these chambers we can't be sure that awakening them would not put them in danger. We also don't understand how the technology works, or how it is controlled. So ripping one of the triangles out of the wall could be catastrophic. In the top left corner of the room there was a minor cave in which may have destroyed some of the pods. None of the team has been able to reach such heights as whoever constructed this room left no stairs, or ladders, or ramps and it is unclear how those in the upper pods would have been awakened without falling to their deaths. The team claims to have several possible plans that they will begin working on before their next report is due."
```
Comment (puckhead73) [inline localisation/english/btr_archaeological_events_l_english.yml:222]: I'd remove the line "So ripping one of the triangles out of the wall could be catastrophic." Sentence reads weird and isn't grammatically correct.

Comment (mstonefield) [reply localisation/english/btr_archaeological_events_l_english.yml:222]: Blame Risker. All those DC PCs are goin to his head

Comment (puckhead73) [reply localisation/english/btr_archaeological_events_l_english.yml:222]: Don't worry about making these changes, reading through these events I think we will want to do a full editing pass, but we can save it for later.

**Review by puckhead73 — APPROVED** [review]
Lots of Localization stuff that I think needs review, but that's a topic we're also going over on Sunday so we can tackle it then.

**Comment by mstonefield** [comment]
We need a designated Grammar Nazi

**Review by maxim-a-k — APPROVED** [review]

**Pull request merged**